commands/app.go

```diff
@@ -153,6 +159,23 @@ To get a linked authority token:
 	return nil
 }
 
+func pathToConfigFile(ctx *cli.Context) (string, error) {
+	if !ctx.IsSet("context") {
```

Comment:
This is probably better:
```suggestion
	if ctx.String("context") == "" {
```